compiler/one-cmds/one-quantize

```diff
@@ -80,27 +83,42 @@ def _get_parser():
     quantization_group.add_argument(
         '--quantized_dtype',
         type=str,
-        help='output quantized data type (supported: uint8, int16, default=uint8)')
+        help='data type of output quantized model (supported: uint8, int16, default=uint8)'
+    )
     quantization_group.add_argument(
         '--granularity',
         type=str,
-        help='quantize granularity (supported: layer, channel, default=layer)')
+        help='quantization granularity (supported: layer, channel, default=layer)')
     quantization_group.add_argument(
         '--input_type',
         type=str,
-        help='input data type (supported: uint8, int16, default=quantized_dtype)')
+        help=
+        'data type of inputs of quantized model (supported: uint8, int16, default=quantized_dtype). Quantize Op wil be inserted at the beginning of the quantized model if input_type is different from quantized_dtype.'
+    )
     quantization_group.add_argument(
         '--output_type',
         type=str,
-        help='output data type (supported: uint8, int16, default=quantized_dtype)')
+        help=
+        'data type of outputs of quantized model (supported: uint8, int16, default=quantized_dtype). Quantize Op will be inserted at the end of the quantized model if output_type is different from quantized_dtype.'
+    )
     quantization_group.add_argument(
-        '--min_percentile', type=str, help='minimum percentile (0.0~100.0, default=1.0)')
+        '--min_percentile',
+        type=str,
+        help=
+        'minimum percentile (0.0~100.0, default=1.0). Algorithm parameter for calibration. This is valid when record mode is percentile.'
+    )
     quantization_group.add_argument(
-        '--max_percentile', type=str, help='maximum percentile (0.0~100.0, default=99.0)')
+        '--max_percentile',
+        type=str,
+        help=
+        'maximum percentile (0.0~100.0, default=99.0). Algorithm parameter for calibration. This is valid when record mode is percentile.'
+    )
     quantization_group.add_argument(
         '--mode',
         type=str,
-        help='record mode (supported: percentile/moving_average, default=percentile)')
+        help=
+        'calibration algorithm for post-training quantization (supported: percentile/moving_average, default=percentile). percentile mode uses the n-th percentiles as min/max values. moving_average mode records the moving average of min/max.'
+    )
```

Comment:
```suggestion
        'calibration algorithm for post-training quantization (supported: percentile/moving_average, default=percentile). Percentile mode uses the n-th percentiles as min/max values. moving_average mode records the moving average of min/max.'
    )
```

Reply:
came from `percentile/moving_average`...
(optional) maybe use 'percentile'/'moving_average' with `'` for argument names...

Reply:
Oh, I see.

Reply:
@seanshpark is right :) I added `'` for argument names.